package.json

```diff
@@ -29,9 +29,9 @@
     "url": "https://github.com/w3c/ghapi/issues"
   },
   "dependencies": {
+    "cli-progress-bar": "1.0.1",
     "express": "4.14.0",
-    "octonode": "0.7.6",
-    "cli-progress-bar": "1.0.0"
+    "octonode": "tripu/octonode"
```

Comment:
Ah. No bug. It's just that there's no updated version of `octonode` on npm. I [asked the maintainer to publish](https://github.com/pksunkara/octonode/pull/248#issuecomment-263916043). In the meantime, to avoid blocking this PR, we can refer to this specific commit.